home/views.py

```diff
@@ -12,24 +12,27 @@
 def index(request):
     # if this is a POST request we need to process the form data
     if request.method == 'POST':
-        # create a form instance and populate it with data from the request:
-        form = InviteForm(request.POST)
-        # check whether it's valid:
-        if form.is_valid():
-            # process the data in form.cleaned_data as required
-            # send John and Nick a slack message with the person's information
-            email = form.cleaned_data["email"]
-            found_us = form.cleaned_data["how_did_you_find_us"]
-            message = f'Somebody just submitted a form!\nEmail: {email}\nHow They Found Us: {found_us}'
-            notify_admins(slack_client, message)
-            # redirect to a new URL:
-            return HttpResponseRedirect('/thank-you/')
+        if settings.SLACK_INVITE_LINK:
```

Comment:
Could you add a `notify_admins` with "Somebody clicked Join." here? Then we can compare this to the number of people actually joining and get a sense of how many people. This might get spammy, but it's not hard to change if it is.